.github/workflows/release_dev_chart.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release Charts
+
+on:
+  workflow_run:
+    workflows: ["unit tests"]
+    types:
+      - completed
+    branches:
+      - dev
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.9.2
+
+      - name: Seed helm deps
+        run: |
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+
+      - name: Run chart-releaser
```

Comment:
The important stuff:
> mark_as_latest: false

don't mark as latest release in github

> CR_RELEASE_NAME_TEMPLATE
add -dev suffix to the release name